dozer/cogs/levels.py

```diff
@@ -284,86 +302,129 @@ async def setrolelevel(self, ctx, role: discord.Role, level: int):
+                """
 
-    @command(aliases=["configurelevels"])
+    @command(aliases=["delrolelevel"])
     @guild_only()
-    @has_permissions(administrator=True)
-    async def configureranks(self, ctx):
-        """Configure dozer ranks:tm: any key word arguments not entered will be treated as default"""
-        args = self.kwarg_parsing(ctx.message.content)  # Parse for kwargs
-
-        xp_min = max(min(int(args.get("min")), 32767), 0) if args.get("min") else 5
-        xp_max = max(min(int(args.get("max")), 32767), 0) if args.get("max") else 15
-        xp_cooldown = max(min(int(args.get("cooldown")), 32767), 0) if args.get("cooldown") else 15
-        try:
-            lvl_up_msgs = ctx.message.channel_mentions[0] if args.get("notify") else None
-            lvl_up_msgs_id = int(ctx.message.channel_mentions[0].id) if args.get("notify") else -1  # The entry is not set to None because issue#195
-        except IndexError:  # Make sure a channel mention is actually attached to the argument
-            raise BadArgument("--notify requires a channel mention!")
-        enabled = False if args.get("disabled") else True  # Travis-Ci this is intentional and not simplifiable
+    @has_permissions(manage_roles=True)
+    async def removerolelevel(self, ctx, role: discord.Role):
+        """Removes a levelrole"""
+        e = discord.Embed(color=blurple)
+        async with ctx.channel.typing():
+            removed = int((await XPRole.delete(role_id=int(role.id))).split(" ", 1)[1])
+            if removed > 0:
+                await self.update_level_role_cache()
+                e.add_field(name='Success!', value=f"{role.mention} was removed from the levels database")
+            else:
+                e.add_field(name='Failed!', value=f"{role.mention} was not found in the levels database!")
+            e.set_footer(text='Triggered by ' + ctx.author.display_name)
+            await ctx.send(embed=e)
+    removerolelevel.example_usage = """
+    `{prefix}removerolelevel level 2 `: Will remove role "level 2" from level roles
+    """
 
+    @group(invoke_without_command=True, aliases=["configurelevels"])
+    @guild_only()
+    @has_permissions(manage_guild=True)
+    async def configureranks(self, ctx):
+        """Configures dozer ranks:tm:"""
+        if ctx.invoked_subcommand is None:
+            raise BadArgument('Invalid config command passed!')
+
+    @configureranks.command()
+    @has_permissions(manage_guild=True)
+    async def xprange(self, ctx, xp_min: int, xp_max: int):
+        """Set the range of a servers levels random xp"""
         if xp_min > xp_max:
             raise BadArgument("XP_min cannot be greater than XP_max!")
-
+        await self._config_guild_setting(ctx, xp_min=xp_min, xp_max=xp_max)
+
+    @configureranks.command()
+    @has_permissions(manage_guild=True)
+    async def setcooldown(self, ctx, cooldown: int):
+        """Set the time between messages before xp is calculated again"""
```

Comment:
Is `cooldown` a number of minutes or seconds? The help message should say which one.